File: hoge.js
a = `def accumulate(combiner, base, n, term):
    previous = term(base)
    for i in range(1, n+1):
        previous = combiner(previous, term(i))
    return previous`

b = `def accumulate(combiner, base, n, term):
    previous = base
    for i in range(1, n+1):
        previous = combiner(previous, term(i))
    return previous`

File: data/hoge.py
import json
from parse_ast import make_ast

def parse(code):
  try:
    return json.dumps({ 'ast': make_ast(code) })
  except Exception as e:
    return json.dumps({ 'ast': { 'error': 'yes', 'why_error?': 'it doesn\'t compile, yo' }})


code = 'previous = combiner(previous, term(i))'
result = parse(code)

print result